docs/competition/teams/process-docs/orientation-and-pit-rules.md

```diff
@@ -0,0 +1,81 @@
+---
+original:
+  authors: Sean Bedford
+  url: https://github.com/srobo/comp-team-coord-docs/ProcessDocs/orientationAndPitRules
+---
+## Welcome to Student Robotics 2017!
+
+## Pit Rules and Guidance
+The following rules must be observed at all times in the pits:
+
+1.	No battery charging. You must hand over your battery charger and spare batteries at reception. During the competition, visit Helpdesk (on the top floor, see map) to swap a discharged battery for a charged one.
+2.	No liquids except drinks to be present on the top floor. The carpet is expensive, and easily damaged by adhesives / paint / solvents.
+3.	No powered tools on the top floor (such as soldering irons). **Power tools should only be used in the downstairs area.**
+4.	Keep your robot in your pit when possible – we may need to find you quickly.
+5.	Please avoid blocking or restricting aisles through pits.
+6.	Please refrain from running any wireless access points, they will only interfere with the AP next to your pit
+
+## Network and Website Details
```

Comment:
Probably not worth documenting the network details here, as we rotate them with each competition, if we run our own network at all.

Reply:
cb34c96 clarifies that this whole page is an example.